intg/src/main/java/org/apache/atlas/model/typedef/AtlasStructDef.java

```diff
@@ -280,6 +280,8 @@ public static class AtlasAttributeDef implements Serializable {
         public static final String    ATTRDEF_OPTION_APPEND_ON_PARTIAL_UPDATE = "isAppendOnPartialUpdate";
         private final String          STRING_TRUE                             = "true";
         private final String          MULTIFIELDS                             = "multifields";
+        private Boolean               isIndexCreated                          = true;
+        private Boolean               isIndexPresent                          = false;
```

Comment:
Please consider using primitive data types, and no need to set `false` to boolean variables, since default value is `false`